src/box-client.ts

```diff
@@ -141,15 +141,15 @@ function getFullURL(defaultBasePath, url) {
  * @returns {Object} The batch API request object
  * @private
  */
-function formatRequestForBatch(params) {
-
+function formatRequestForBatch(params: Record<string, string>) {
```

Comment:
Is `params` always shaped like `string: string`? Is `params.body` not `string: object` in this case?